clients/client-shell/cmds/validate-json/validate.go

```diff
@@ -0,0 +1,71 @@
+// Package validatejson impliments validate-json command
+package validatejson
+
+import (
+	"errors"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/taskcluster/taskcluster/v44/clients/client-shell/cmds/root"
+	js "github.com/xeipuuv/gojsonschema"
+)
+
+var (
+	Command = &cobra.Command{
+		Use:   "validate-json <json-schema> <json-file>",
+		Short: "Validate json file by provided json-schema",
+		RunE:  validate,
+	}
+
+	log = root.Logger
+)
+
+func init() {
+	root.Command.AddCommand(Command)
+}
+
+// Takes json-schema & json as input which should be in the following 2 formats
+// 1. https://community-tc.services.mozilla.com/references/schemas/<some-schema>.json
+// 2. file:///home/user/<some-schema>.json
+func validate(cmd *cobra.Command, args []string) error {
+	schema := schemaLoader(args[0])
```

Comment:
If I pass no arguments, it fails with:

```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/taskcluster/taskcluster/v44/clients/client-shell/cmds/validate-json.validate(0x19c98c0, 0x19eef90, 0x0, 0x0, 0x0, 0x0)
	/Users/john/src/taskcluster/clients/client-shell/cmds/validate-json/validate.go:32 +0x36f
github.com/spf13/cobra.(*Command).execute(0x19c98c0, 0x19eef90, 0x0, 0x0, 0x19c98c0, 0x19eef90)
	/Users/john/.gvm/pkgsets/go1.13.7/global/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0xc0000c4280, 0x0, 0x8a400, 0xc000098058)
	/Users/john/.gvm/pkgsets/go1.13.7/global/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/john/.gvm/pkgsets/go1.13.7/global/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
main.main()
	/Users/john/src/taskcluster/clients/client-shell/main.go:15 +0x32
```

Looking at the cobra docs, there are ways to specific [Positional arguments](https://cobra.dev/#positional-and-custom-arguments), but the existing `from-now` command takes a more direct approach:

https://github.com/taskcluster/taskcluster/blob/26463ffcf7fd416c8498f5fff3c56af436e8211a/clients/client-shell/cmds/from-now/from_now.go#L28-L30

Reply:
That is a good catch, I'll add the positional arguments check.